contracts/seed/Seed.sol

```diff
@@ -274,16 +266,14 @@ contract Seed {
     /**
       * @dev         Returns funding tokens to user.
     */
-    function retrieveFundingTokens() public allowedToRetrieve returns(uint256) {
+    function retrieveFundingTokens() external allowedToRetrieve returns(uint256) {
```

Comment:
I would remove allowedToRetrieve and move its contents to here

Reply:
yes, good idea. Thanks

Reply:
Done